### DISCUSSION TITLE
Player Build/Break

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#1325] [Q&A] [answered]
HEY THERE,
This plugin is GR8 but i got a question that how do i let players build on land without claiming it
when players try to break or place blocks it says u must claim land to break/place block here.... smthin
how do i fix it
if i have to fix it using cmnd thn can u plz reply with the command or if i have to config it from config.yml thn plz reply with the line number in config.yml or if any other... plz help me :)

The plugin version is the latest version and game version is 1.16.5

**Comment by bigpresh** [comment, marked as answer]
You'll want to use `Survival` rather than `Creative` or `SurvivalRequiringClaims` under claims.mode in your config.yml - full details at https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on

**Comment by ghost** [comment]
So does it mean if I change the mode to Survival for the world Survival rhn ppl can build/break blocks without claiming the land @bigpresh

**Reply by RoboMWM** [reply]
yes.

**Reply by ghost** [reply]
k I'll try

**Reply by ghost** [reply]
Hey it doesnt work @RoboMWM

**Reply by bigpresh** [reply]
Please share your config file, then, so we can see what you've put in there, because that's exactly how it works.

You said "change the mode to Survival for the world Survival" - if the world you're trying to protect is named "Survival" then yes, you'd use e.g.:

```
  Claims:
    Mode:
      Survival: Survival
```

Straight from the config on my server, where the default world is named just "world", and it works exactly as you're looking for:

```
  Claims:
    Mode:
      world: Survival
      world_the_end: Survival
      world_nether: Survival
```

**Comment by ghost** [comment]
Claims:
    Mode:
      Creative: Survival
      world: Disabled
      world_the_end: Disabled
      BedWars: Disabled
      Hub: Survival
      world_nether: Disabled
      Parkour: Disabled
      BattleRoyale_lobby: Disabled
      BattleRoyale: Disabled
      Survival: Survival
      BW_lobby: Disabled
      PvP: Creative

these are the worlds in my server and i want ppl to build without claiming
land

**Reply by bigpresh** [reply]
And on which world are you finding that players can't build outside of claimed areas?  From that config, the world named `PvP` is the only one where GP should be preventing people from building stuff outside of claims.

**Comment by ghost** [comment]
ik but i've kept Survival claims but still it keeps on saying u need to
claim that land first to build

**Reply by RoboMWM** [reply]
Try changing the option while the server is off. 

If that still doesn't work - how is the world loaded in? With multiverse? Is it loaded at startup or later on elsewhere?

**Comment by ghost** [comment]
aight lemme try

**Reply by ghost** [reply]
hey i see that players can place blocks but cant break
is that because of EssentialX or GP

**Reply by RoboMWM** [reply]
Might be another plugin. Try with just GP. Then add Essentials, etc.

**Reply by ghost** [reply]
yeah ill try removing essentials and letting GP be

**Reply by ghost** [reply]
Hey i tried removing essentials but still wont work
can u help me out plz :( @RoboMWM 
Can u just Dm me on Discord AloneBoy#6982

**Reply by RoboMWM** [reply]
got other plugins?

**Reply by ghost** [reply]
nope

**Reply by RoboMWM** [reply]
Please paste output of `/version`, `/version GriefPrevention`, `bukkit:pl`

**Reply by ghost** [reply]
ok ill do it soon :D

**Reply by gold0s** [reply]
i know this was awhile ago but was this ever fixed i have the same problem :0

**Reply by RoboMWM** [reply]
He mentioned he reinstalled in the following comment and that worked. So not sure for you, but it's likely a plugin conflict.

**Comment by ghost** [comment]
yeah i re-installed it and it worked
